dash/frontend/src/app/modules/private/pages/falco/falco-events-list/falco-events-list.component.ts

```diff
@@ -216,16 +216,20 @@ export class FalcoEventsListComponent implements OnInit {
       });
   }
 
+  truncateString(rawString) {
```

Comment:
Since it looks like you are just trying to use this for display purposes, you could accomplish this with a CSS class, and that way it would use as many characters as it could fit rather than exactly 17 (Assuming of course the goal isn't to have exactly 17 characters)

.some-class {
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
}